src/mediarithmics/api/core/plugin/PluginPropertyInterface.ts

```diff
@@ -72,6 +75,12 @@ export interface AssetFileProperty
   value: AssetFilePropertyResource;
 }
 
+export interface AssetFolderProperty
```

Comment:
I'm not sure if it's Github but the formatting seems to be wrong

Reply:
Also, can you update `AssetFileProperty` so that it is based on property_type: `ASSET_FILE` instead of `ASSET`?